components/tag-list/test/tag-list.test.js

```diff
@@ -91,6 +91,24 @@ describe('d2l-tag-list-item', () => {
 		});
 	});
 
+	describe('display text', () => {
+		it('should default to tagContent', async() => {
```

Comment:
```suggestion
		it('should use text for tag-list-item', async() => {
```
Looks like this isn't really accurate anymore, maybe something like this?